source/docs/quiche_integration.md

```diff
@@ -35,3 +35,8 @@ When the aggregated buffered bytes goes above high watermark, its registered net
 As to Http::StreamEncoder::encodeHeaders()/encodeTrailers(), the accounting is done differently between Google QUIC and IETF QUIC:
  * In Google QUIC, encodeHeaders()/encodeTrailers() check the buffer size increase on header stream before and after writing headers/trailers. In QuicSession::OnCanWrite(), may drain header stream send buffer, so there we also check send buffer size decrease on header stream.
  * In IETF QUIC, encodeHeaders()/encodeTrailers() check the buffer size increase on the corresponding data stream which is similar to encodeData(). The buffered headers/trailers are only drained via QuicStream::OnCanWrite() so there is no need to check QuicSession::OnCanWrite.
+
+### Debugging
```

Comment:
While you are in here do you mind also documenting the nghttp2 env var? https://github.com/envoyproxy/envoy/blob/main/source/common/http/http2/nghttp2.cc

Would it be worth it to standardize the names a bit also? I don't feel strongly but might be nice.

And finally, if you search fro ENVOY_NGHTTP2_TRACE you will see it's documented a few different places. Might be nice to consolidate and/or cross reference.

Reply:
I intentionally picked a slightly different name because `ENVOY_NGHTTP2_TRACE` is a bool whereas `ENVOY_QUICHE_VERBOSITY` is an int. What do you think?

I just modified the documentation that referenced `ENVOY_NGHTTP2_TRACE` to also reference `ENVOY_QUICHE_VERBOSITY`.

Reply:
Sure sounds good.